### ISSUE TITLE
Would you ever consider being my professional mentor?

### ISSUE THREAD

**Issue by CrystalZenyth** [KittyGiraudel/ama#126]
Hi Kitty,

I am a Web Accessibility Engineer with ZenythGroup. I spend my days auditing websites against WCAG, remediating websites to meet WCAG, building new websites following WCAG and accessibility best practices and project managing/developing for a number of internal software projects. What I am missing in my life is someone to help guide me into what I should be learning and mastering? I am a lead at the company I work for but don't always feel like I know what I am doing. I took the IAAP WAS cert in December which was great but this year I'd like to work on a software project management cert but not sure which is the best or if that is the best direction for me. I am almost an expert in HTML, but still feel intermediate to beginner in CSS and I am able to write to JavaScript I need to fix minor website issues but nothing complex. Anywho, enough about me, I am very impressed by your work and we use your a11y dialog pattern in all of our builds. You have a killer website with so many great projects! I would kill for the opportunity to meet with you and pick your brain. Would you make some time to meet with me?

**Comment by KittyGiraudel** [comment]
Hi Crystal, nice to meet you! ✨

Honestly your background is quite impressive so I’m not sure you’d need a mentor at all — let alone me! That being said, I’d be happy to meet you. Let’s connect on LinkedIn and find some time for a call. :)
